docs/specs/user-export.md

```diff
@@ -0,0 +1,284 @@
+# User Export
+
+User Export allows the developer to bulk export all users from Authgear to a file.
+
+## About User Export
+
+- It is not a synchronous operation. The export is created and runs in the background. The developer can query the status of it.
+
+## Create an export
+
+The endpoint is `POST /_api/admin/users/export`.
+
+- The endpoint requires Admin API JWT token to access.
+- This endpoint is added to Admin API server.
+- It is not part of the GraphQL API.
+- At most 1 running export at any given moment for a project.
+- The rate limit of creating a export is 24 per 24h. This means failed requests do not count.
```

Comment:
Is this rolling 24 hours or refresh at every 00:00 UTC?

Reply:
It is rolling. And it will be implemented using our existing rate limit mechanism.

Reply:
Do we allow controlling it in the config?